manim/mobject/matrix.py

```diff
@@ -141,7 +142,7 @@ def __init__(
             self.add_background_rectangle()
 
     def matrix_to_mob_matrix(self, matrix):
-        return np.vectorize(self.element_to_mobject)(
+        return np.vectorize(self.element_to_mobject, otypes=[object])(
```

Comment:
What does this do?

Reply:
Initially, OpenGL didn't work, and I had to make changes to this function but @charburgx found a solution, which is to set the output type to object to make it compatible with OpenGL. Maybe @charburgx can elaborate more.